.github/workflows/changelog.yml

```diff
@@ -0,0 +1,77 @@
+name: Centralized EdgeX Changelog Generation
+
+on:
+  push:
+    branches:
+      - changelog-generator
+  schedule:
+    - cron: '0 0 * * 0' # Run every Sunday at 00:00
+  workflow_dispatch: # Allow manual triggering
```

Comment:
Will committers be able to manually trigger?

Reply:
Correct